src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -230,7 +230,7 @@ public async ValueTask<object> ReceiveOneAsync(TimeSpan? max = null, Cancellatio
             CancellationToken cancellationToken = default)
         {
             var (success, messageEnvelope) = TryReceiveOneAsync(max, cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.